persistence/src/main/java/edu/unc/lib/dl/util/TombstonePropertySelector.java

```diff
@@ -37,9 +37,9 @@
 public class TombstonePropertySelector extends SimpleSelector {
 
     List<Property> permittedPredicates = Arrays.asList(
```

Comment:
Don't think this will do quite what we want, since the Binary will have the server managed triples. There will probably need to be a step that remaps the original properties to the cdr namespace versions